### PR TITLE
Minor README edit

### PR DESCRIPTION
**Pull request by TheTechnician27**
Made minor changes to the README. Most notably made clear that it's free and open-source since there's no mention of the GPL in the README and changed less to fewer because "less" isn't correct in that context.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ Development builds are also available from [our website](https://pcsx2.net/downl
 
 - Make sure to update your operating system, drivers, and DirectX (if applicable) to ensure you have the best experience possible. Having a newer GPU is also recommended so you have the latest supported drivers.
 
-- Because of copyright issues, and the complexity of trying to work around it, you need a BIOS dump extracted from a legitimately-owned PS2 console to use the emulator. For more information about the BIOS and how to get it from your console, visit [this page](https://pcsx2.net/config-guide/official-english-pcsx2-configuration-guide.html#Bios).
+- Because of copyright issues—and the complexity of trying to work around it—you need a BIOS dump extracted from a legitimately-owned PS2 console to use the emulator. For more information about the BIOS and how to get it from your console, visit [this page](https://pcsx2.net/config-guide/official-english-pcsx2-configuration-guide.html#Bios).
```
Comment (lightningterror) [inline README.md]: This line didn't need changing.

Comment (pgert) [reply README.md]: ... at least, it should be " - ", not "-" (spaces between the dash sign).

**Comment by lightningterror** [comment]
@TheTechnician27 can you update the PR ? If not then I'm going to have to close it since it's really low priority.

File: README.md [hunk 2]
```diff
@@ -35,9 +35,9 @@ Development builds are also available from [our website](https://pcsx2.net/downl
...
 
-- PCSX2 mainly takes advantage of 2 CPU cores. As of [this commit](https://github.com/PCSX2/pcsx2/commit/ac9bf45) PCSX2 can now take advantage of more than 2 cores using the MTVU speedhack. This can be a significant speedup on CPUs with 3+ cores, however on GS-limited games (or on CPUs with less than 2 cores) it may be a slowdown.
+- PCSX2 mainly takes advantage of 2 CPU cores. As of [this commit](https://github.com/PCSX2/pcsx2/commit/ac9bf45), PCSX2 can now take advantage of more than 2 cores using the MTVU speedhack. This can be a significant speedup on CPUs with 3+ cores, however on GS-limited games (or on CPUs with fewer than 2 cores), it may be a slowdown.
```
Comment (pgert) [inline README.md]: This added comma is wrong, isn't it? Puts the clause out of focus.

**Pull request closed without merging**